services/galley/src/Galley/Cassandra/Conversation.hs

```diff
@@ -191,6 +191,32 @@ getConversation conv = do
       <*> UnliftIO.wait cdata
   runMaybeT $ conversationGC =<< maybe mzero pure mbConv
 
+getGlobalTeamConversation :: TeamId -> Client (Maybe Conversation)
+getGlobalTeamConversation tid = do
+  (\c -> c {convLocalMembers = [], convRemoteMembers = []})
+    <$$> getConversation (globalTeamConv tid)
+
+createGlobalTeamConversation :: Local TeamId -> UserId -> Client Conversation
+createGlobalTeamConversation tid uid = do
+  createConversation (globalTeamConv <$> tid) nc
+  where
+    nc =
+      NewConversation
+        { ncMetadata =
+            ConversationMetadata
+              { cnvmType = GlobalTeamConv,
+                cnvmCreator = uid,
+                cnvmAccess = [LinkAccess],
```

Comment:
I think this should be `cnvmAccess = [InviteAccess]`, as users should be able to join only by someone else adding them (namely, the team admin). `LinkAccess` means it is sufficient to know the conversation ID.

Reply:
Yeah, I thought you would join by `convId`, by querying the `GET` endpoint for global team convs. This is sadly not specified in the spec either.

Reply:
Then the list is `[InviteAccess, LinkAccess]`, right?